README.md

```diff
@@ -1,11 +1,10 @@
-# govuk_publishing_components
-A gem to document components in GOV.UK frontend applications
+# GOV.UK Publishing Components
+
+A gem to document components in GOV.UK frontend applications.
 
 ## Usage
-How to use my plugin.
 
-## Installation
-Add this line to your application's Gemfile:
+Add this line to your application's Gemfile in the development and test groups:
```

Comment:
As someone new to this, I don't know what the "development and test groups" are, so I'd be stuck on this step.